app/rdf/queries.ts

```diff
@@ -349,9 +356,29 @@ const getCubeDimensionValuesWithLabels = async ({
     dimensionValueNamedNodes.length === 0 &&
     dimensionValueLiterals.length === 0
   ) {
-    console.warn(
-      `WARNING: dimension with NO values <${dimension.path?.value}>`
-    );
+    const resolvedDimensionValues = await loadDimensionValues({
+      datasetIri: cube.term,
+      dimensionIri: dimension.path,
+    });
+
+    if (resolvedDimensionValues.length) {
+      const unpackedDimensionValues = resolvedDimensionValues.map(
+        (d) => d.value
+      );
+
+      dimensionValueNamedNodes = filterCubeDimensionValuesByTermType({
+        values: unpackedDimensionValues,
+        termType: "NamedNode",
+      }) as NamedNode[];
+      dimensionValueLiterals = filterCubeDimensionValuesByTermType({
+        values: unpackedDimensionValues,
+        termType: "Literal",
+      }) as Literal[];
```

Comment:
Nit: Seems like the logic is repeated from above no ? Maybe it could be expressed in the way "there is two places to take the dimensionValues, either in dimension.in or through loadDimensionValues ?

```
const loadDimensionValues = async () => {
	  const dimensionLoaders = [() => dimension.in, () => loadDimensionValues({ })]
	  
	  for (let dimensionLoader of dimensionLoaders) {
	    const dimensionValues = await dimensionLoader()
	    // then groupBy named nodes and literals
	    const { NamedNode: namedNodes = [], Literal: literals = [] } = groupBy(dimensionValues, x => x.termType)
	    if (namedNodes.length || literals.length) {
	      return { namedNodes, literals }
	    }  
	  }
          console.warn('no values...')
}
```

(untested)